docs/user/alerting/troubleshooting/alerting-common-issues.asciidoc

```diff
@@ -73,7 +73,23 @@ By default, only users with a `superuser` role can query the experimental[] {kib
 
 *Solution*
 
-Query for a list of rule ids, bucketed by their execution times:
+By default, rules have a `5m` timeout. Rules that run longer than this timeout are automatically cancelled to prevent them from consuming too much of {kib}'s resources. When a rule times out, you will see this error in the {kib} logs:
```

Comment:
**question:** should we mention what happens to the actions when the rule times out? (that they all get dropped, and there's no partial sending)

Reply:
Good catch. Added some words for this here: https://github.com/elastic/kibana/pull/128677/commits/ac80594987074138be9b34c05940908ce426bcbe